addons/calendar/static/src/views/attendee_calendar/attendee_calendar_controller.xml

```diff
@@ -43,5 +43,8 @@
                 </t>
             </div>
         </xpath>
+        <xpath expr="//t[@t-elif]//span[hasclass('badge bg-100 rounded')]" position="attributes">
+            <attribute name="t-if" add="!env.isSmall" separator="and"/>
+        </xpath>
```

Comment:
It will definitely break in future if the class name evolve because their are not specific to week numbers (generic class naemes). Why not either propose the fix in web.CalendarController or add an identifier in the parent to be able to target it easily ?

Reply:
Hello @arj-odoo,
I did the changes but forgot to ping you sorry :sweat: 
can you please have a look